app/collections/favorites-button.cjsx

```diff
@@ -89,6 +89,9 @@ module?.exports = React.createClass
         className="favorites-button"
         type="button"
         onClick={@toggleFavorite}>
-        <i className={"fa fa-heart#{if favorited then '' else '-o'}"} />
+        <i className="
+          fa fa-heart#{if @state.favorited then '' else '-o'}
```

Comment:
Ah sorry I should've checked that more closely in the merge, I'll push that up with the other changes in one sec